src/Backend.cpp

```diff
@@ -36,13 +37,17 @@ bool HABackend::Connect(string url, string token)
 bool HABackend::Start()
 {
   if (wc == nullptr) {
-    cerr << "Do a damned connect first" << endl;
+    cerr << "We expect that you'd do a Connect() first." << endl;
     return false;
   }
-  cerr << "In start" << endl;
+  loaded = false;
+  std::unique_lock<std::mutex> lck(load_lock);
   ha = std::thread(&HABackend::threadrunner, this);
-  cerr << "Thred started" << endl;
   ha.detach();
+  while (!loaded) {
+    load_cv.wait(lck);
+    usleep(200);
```

Comment:
putting the usleep before the wait could shave 200 microseconds off of our startup time :D